### ISSUE TITLE
[Bug] dev.2.1.1-rc.4 DB Migration Fail

### ISSUE THREAD

**Issue by thorin8k** [zurdi15/romm#512]
**RomM version**
`dev.2.1.1-rc.4` latest develoment tag

**Describe the bug**
App wont start because of migrations

**To Reproduce**
1. Run the application using sqlite and latest docker image
2. Application wont start
3. Logs show alembic update error

**Expected behavior**
System should start

**Additional context**
Tried with an existing database and a new created one.

Log message:

```
Running upgrade 2.0.0 -> 0009_models_refactor, empty message
Traceback (most recent call last):
  File "/backend/lib/python3.10/site-packages/alembic/operations/batch.py", line 689, in drop_constraint
    const = self.named_constraints.pop(const.name)
KeyError: 'PRIMARY'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/backend/bin/alembic", line 8, in <module>
    sys.exit(main())
  File "/backend/lib/python3.10/site-packages/alembic/config.py", line 591, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/backend/lib/python3.10/site-packages/alembic/config.py", line 585, in main
    self.run_cmd(cfg, options)
  File "/backend/lib/python3.10/site-packages/alembic/config.py", line 562, in run_cmd
    fn(
  File "/backend/lib/python3.10/site-packages/alembic/command.py", line 378, in upgrade
    script.run_env()
  File "/backend/lib/python3.10/site-packages/alembic/script/base.py", line 576, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/backend/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
    module = load_module_py(module_id, path)
  File "/backend/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
    spec.loader.exec_module(module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/backend/alembic/env.py", line 82, in <module>
    run_migrations_online()
  File "/backend/alembic/env.py", line 76, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/backend/lib/python3.10/site-packages/alembic/runtime/environment.py", line 868, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/backend/lib/python3.10/site-packages/alembic/runtime/migration.py", line 622, in run_migrations
    step.migration_fn(**kw)
  File "/backend/alembic/versions/0009_models_refactor.py", line 20, in upgrade
    with op.batch_alter_table("platforms", schema=None) as batch_op:
  File "/usr/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/backend/lib/python3.10/site-packages/alembic/operations/base.py", line 383, in batch_alter_table
    impl.flush()
  File "/backend/lib/python3.10/site-packages/alembic/operations/batch.py", line 159, in flush
    fn(*arg, **kw)
  File "/backend/lib/python3.10/site-packages/alembic/operations/batch.py", line 700, in drop_constraint
    raise ValueError("No such constraint: '%s'" % const.name)
ValueError: No such constraint: 'PRIMARY'
```

**Comment by thorin8k** [comment]
Rolling back to dev.2.1.1-rc.1 the application was able to start up again with the existing db.

**Comment by gantoine** [comment]
Ok I think I have a fix for this under #515, zurdi will likely take a look at it after the holidays.
